pitch.md

```diff
@@ -23,38 +89,174 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+such as arithmetic, nested control flow, or see implementation details.
+
+I don't think there should be anything in between (of medium difficulty).
+
 
 ## Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+As I mentioned there are definitely already DSLs in this domain:
+ * [WhenIsGood](http://whenisgood.net/)
+ * [When2Meet](http://www.when2meet.com)
+
+These are indeed DSLs because they are computerized schedulers
+with a specific syntax (painting over certain blocks), and associated
+semantics (painted blocks are free blocks).
+
+These DSLs are quite minimalistic (i.e. when they run, they only support the
+intersection operation for the sets of times from each individual -- as far
+as I can tell).  I am also looking at some of the features provided for
+[Premium Users](http://whenisgood.net/PremiumInfo).
+
+My solution to the scheduling problem might be a bit more
+ambitious in the functionality it hopes to provide.  This does allow for more
+expressiveness, which is nice, but expressiveness is a double edged sword,
+I will need to take care during design to avoid feature creep and extra complexity
+for users.  Perhaps I will find that the existing websites' simplicity is
+more desirable than the benefits of the expressiveness I aim to provide (but
+I hope not!).
+
+I think that they address the need pretty well at the present, but I think that
+my DSL could cut down on repetition and allow users to better express themselves.
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+The main reason I finally settled on this project was that I felt it had the best
+balance of **language** design to systems programming of any of my potential
+choices.  As I mentioned in the previous assignment, the underlying internal
+representation could be represented with just a [protocol
+buffer](https://en.wikipedia.org/wiki/Protocol_Buffers) for each individual.
+
+Armed with the parsing tools from the Picobot lab, the work of writing a 
+parser to take the syntax of my language to the internal representation should 
+be overshadowed by the work of designing the best interface/fluency of the
+language for the user's sake.
+
+Based on Daniel Sonner's helpful advice, I have decided not to focus on the
+algorithms to find an optimal schedule given a set of constraints, which could
+potentially be *very* interesting, so that I can focus the lion's share of
```

Comment:
I think this is a good idea&mdash;the CSP behind the language could be very complicated given enough expressiveness. Finding the balanace is key.